lib/main.d.ts

```diff
@@ -0,0 +1,286 @@
+import * as Swagger from 'swagger-schema-official'
```

Comment:
Why does it require this module?

Reply:
The type of an api (used for `ow.routes`) is:
```
interface Api {
        namespace: string;
        gwApiActivated: boolean;
        tenantId: boolean;
        gwApiUrl: string;
        apidoc: Swagger.Spec;
    }
```

The field `apidoc` contains the swagger of the Api. 
Rather than redefining the types for swagger, I used the package swagger-schema-official which only contains the type definitions for Swagger.
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/swagger-schema-official